### PR TITLE
Add datastore backups

### PR DESCRIPTION
**Pull request by Rey-Lane**
Added new parameters to datastore schemas;
Created new schema for backup parameters;
Provided a new endpoint to update datastore backups parameters.

### CODE REVIEW

File: datastore.go [hunk 1]
```diff
@@ -374,3 +382,31 @@ func (api *API) PasswordDatastore(ctx context.Context, datastoreID string, opts
 
 	return result.Datastore, nil
 }
+
+func (api *API) BackupsDatastore(ctx context.Context, datastoreID string, opts DatastoreBackupsOpts) (Datastore, error) { //nolint
```
Comment (Gogen120) [inline datastore.go:386]: You should add a docstring for this method

**Comment by Gogen120** [comment]
Also, you need to add test for a new endpoint to the datastore_test.go

File: datastore.go (hunk 1 above)
Comment (Gogen120) [reply datastore.go:386]: Also, you need to add tests for a new endpoint to the datastore_test.go

Comment (Gogen120) [reply datastore.go:387]: Also, it would be great if you add some tests for the function to the `datastore_test.go`

Comment (Rey-Lane) [reply datastore.go:387]: Done

**Pull request merged**